rest-api/offline/public_metrics_export.py

```diff
@@ -3,74 +3,99 @@
 from dao.database_utils import replace_years_old
 from participant_enums import EnrollmentStatus, OrderStatus, PhysicalMeasurementsStatus
 from participant_enums import Race, QuestionnaireStatus
+from participant_enums import TEST_EMAIL_PATTERN, TEST_HPO_NAME
 
-# TODO(calbach): consider whether we need to filter down to consented
-# individuals only.
 
 def _questionnaire_metric(name, col):
   """Returns a metrics SQL aggregation tuple for the given key/column."""
   return (
       name,
       """
-      SELECT {}, SUM(1)
+      SELECT {col}, COUNT(*)
       FROM participant_summary
+      WHERE {filter_test_sql}
       GROUP BY 1;
-      """.format(col),
+      """.format(col=col, filter_test_sql=_FILTER_TEST_SQL),
       lambda v: QuestionnaireStatus.lookup_by_number(v).name
   )
 
+_FILTER_TEST_SQL = """
+(NOT participant_summary.email LIKE '{test_email}'
```

Comment:
Any reason not to use bound parameters here like we are elsewhere?

It doesn't matter provided we don't have weird characters in the e-mail or HPO value, but it'd make me feel slightly safer to do LIKE :test_email

Reply:
Sure - done.